front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -134,6 +134,8 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
 
   const { process_type, publication_status } = project.attributes;
 
+  const isArchivedProject =
```

Comment:
For consistency with `ProjectInfoSideBar.tsx`, I suggest to use variable name `isProjectArchived` instead of `isArchivedProject`.

Reply:
Ah good catch - I'll change it. Thanks :)